org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -979,3 +983,41 @@ SsoCnbAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref cnbAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoScicompIncludeDccCollab:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-scicomp-include-dcc-collab'
+  StackDescription: 'SSO: role to access INCLUDE DCC project resources at CHOP'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref ScicompAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref scicompIncludeDccCollabGroup
+    permissionSetName: 'IncludeDccCollab'
+    sessionDuration: 'PT12H'
+    inlinePolicy: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Deny",
+            "Principal": "*",
+            "Action": [
+                "s3:GetObject*",
+                "s3:PutObject*",
+                "s3:DeleteObject*"
+            ],
+            "Resource": "arn:aws:s3:::*",
+            "Condition": {
+              "StringNotLike": {
+                "s3:ResourceAccount": "<CHOP AWS Account ID>",
```

Comment:
@BrunoGrandePhD could you please provide the CHOP AWS account number?

Reply:
You might want to create three statements, one for each account. I'll review the PR once the account numbers are added.

Reply:
done